content/automate/ManageIQ/Service/Provisioning/StateMachines/ServiceProvision_Template.class/__methods__/update_serviceprovision_status.rb

```diff
@@ -2,28 +2,57 @@
 # Description: This method updates the service provision status.
 # Required inputs: status
 #
+module ManageIQ
+  module Automate
+    module Service
+      module Provisioning
+        module StateMachines
+          module ServiceProvision_Template
```

Comment:
@mkanoor @pkomanek It is an internal naming issue, so we have to leave the name as ServiceProvision_Template.